tests/point_cloud/test_workingcopy.py

```diff
@@ -1244,3 +1246,28 @@ def test_working_copy_conflicting_extension(cli_runner, data_archive, requires_p
         r = cli_runner.invoke(["status"])
         assert r.exit_code == INVALID_OPERATION
         assert "More than one tile found in working copy with the same name" in r.stderr
+
+
+def test_working_copy_reflink(cli_runner, data_archive, check_tile_is_reflinked):
+    if is_windows:
+        # Nothing to test, so let's mark test will show as "passed".
```

Comment:
same here...

Reply:
Yeah I was unsure what to do here: wasn't sure if I should try to use pytest.skip to mean "functionality is untested" and just a pass to mean "nothing to test". But I think you're right, pytest.skip makes more sense here.